assets/js/components/surveys/SurveyQuestionSingleSelectChoice.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * SurveyQuestionSingleSelectChoice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Radio from '../Radio';
+import { TextField, Input } from '../../material-components';
+import VisuallyHidden from '../VisuallyHidden';
+
+const MAXIMUM_CHARACTER_LIMIT = 100;
+
+const SurveyQuestionSingleSelectChoice = ( {
+	value,
+	setValue,
+	writeIn,
+	setWriteIn,
+	choice,
+} ) => {
+	const { answer_ordinal, text, write_in } = choice; // eslint-disable-line camelcase
+	const isChecked = value === answer_ordinal; // eslint-disable-line camelcase
+	const uniqueID = `googlesitekit-survey__multi-select-${ answer_ordinal }-${ text }`; // eslint-disable-line camelcase
+
+	return (
+		<div className="googlesitekit-single-select__choice">
+			<Radio
+				id={ text }
```

Comment:
Why is `text` the ID for this? Couldn't text be multiple words with punctuation? HTML ID attributes can't have whitespace so that'd cause errors.

Reply:
TIL this!  :exploding_head: 

But other attributes can. Makes sense!

Reply:
Used same fix as in other PR. Replacing whitespace with dashes